stan/math/prim/mat/prob/lkj_cov_lpdf.hpp

```diff
@@ -58,12 +56,11 @@ return_type_t<T_y, T_loc, T_scale, T_shape> lkj_cov_lpdf(
 }
 
 template <typename T_y, typename T_loc, typename T_scale, typename T_shape>
-inline typename boost::math::tools::promote_args<T_y, T_loc, T_scale,
-                                                 T_shape>::type
-lkj_cov_lpdf(const Eigen::Matrix<T_y, Eigen::Dynamic, Eigen::Dynamic>& y,
-             const Eigen::Matrix<T_loc, Eigen::Dynamic, 1>& mu,
-             const Eigen::Matrix<T_scale, Eigen::Dynamic, 1>& sigma,
-             const T_shape& eta) {
+inline promote_args_t<T_y, T_loc, T_scale, T_shape> lkj_cov_lpdf(
```

Comment:
I think we can replace all of these `promote_args_t` to be `return_type_t`. May even be good to take it a step further and make `promote_arg_t` into an internal namespace within the `return_type.hpp`. Then I think it would be easier to see that `promote_args_t` is only used internally for `return_type_t`

Reply:
Could you open a new issue about this (or I could try to do it)?

Reply:
Sure I'll make an issue